sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShuffleBufferCatalog.scala

```diff
@@ -190,15 +190,11 @@ class ShuffleBufferCatalog(
    * Lookup the shuffle buffer that corresponds to the specified shuffle buffer ID and acquire it.
    * NOTE: It is the responsibility of the caller to close the buffer.
    * @param id shuffle buffer identifier
+   * @param unspill if true, unspill the buffer to device storage
    * @return shuffle buffer that has been acquired
    */
-  def acquireBuffer(id: ShuffleBufferId): RapidsBuffer = {
-    val buffer = catalog.acquireBuffer(id)
-    // Shuffle buffers that have been read are less likely to be read again,
-    // so update the spill priority based on this access
-    val spillPriority = SpillPriorities.getShuffleOutputBufferReadPriority
-    buffer.setSpillPriority(spillPriority)
-    buffer
+  def acquireBuffer(id: ShuffleBufferId, unspill: Boolean = false): RapidsBuffer = {
+    RapidsBufferCatalog.acquireBuffer(id, unspill)
```

Comment:
have you found that the "less likely" piece here is true for the common case? I see we are not updating the priority anymore.

Reply:
Left the priority update in for now.